packages/asc-ui/src/components/LinkList/LinkListItemStyle.ts

```diff
@@ -4,33 +4,47 @@ import { color, svgFill } from '../../utils'
 import IconStyle from '../Icon/IconStyle'
 import TypographyStyle from '../Typography/TypographyStyle'
 
-export default styled.li`
+export type Props = {
+  inverted?: boolean
```

Comment:
What if we want red links? Would pass the color as a prop

Reply:
refactored